runelite-client/src/main/java/net/runelite/client/plugins/alchemicalhydra/AlchemicalHydraPlugin.java

```diff
@@ -56,19 +56,19 @@
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 import net.runelite.client.plugins.PluginType;
-import net.runelite.client.plugins.alchemicalhydra.Hydra.AttackStyle;
+import net.runelite.client.plugins.alchemicalhydra.AlchemicalHydra.AttackStyle;
 import net.runelite.client.ui.overlay.OverlayManager;
 
 @PluginDescriptor(
 	name = "Alchemical Hydra",
 	description = "Show what to pray against hydra",
-	tags = {"Hydra", "Lazy", "4 headed asshole"},
+	tags = {"alchemical", "hydra", "boss", "pvm"},
 	type = PluginType.PVM,
 	enabledByDefault = false
 )
 @Slf4j
 @Singleton
-public class HydraPlugin extends Plugin
+public class AlchemicalHydraPlugin extends Plugin
```

Comment:
Can you elaborate? All I see in both plugins is the enabledByDefault = false in the PluginDescriptor annotation.

Reply:
PluginManager creates config keys for what plugins are/aren't enabled from class names. It's not that big of a problem though; I only decided to comment on that because I've got a grudge against small hydras for stealing the package/class name I originally had